firebase-firestore/src/androidTest/java/com/google/firebase/firestore/CountTest.java

```diff
@@ -34,6 +39,29 @@ public void tearDown() {
     IntegrationTestUtil.tearDown();
   }
 
+  @Test
+  public void testCountQueryEquals() {
```

Comment:
testCountQueryEquals probably makes more sense to be located in AggregateQueryTest since it's a unit test for methods in AggregateQuery.